### ISSUE TITLE
Add WebAuthn to the default `doc-valid-idents` list to prevent `doc_markdown` from being triggered

### ISSUE THREAD

**Issue by zacknewman** [rust-lang/rust-clippy#17539]
### Summary

[WebAuthn](https://www.w3.org/TR/webauthn-3/) is the specification that describes how public-key-based credentials (e.g., passkeys) can work on the web. It would be nice if it could be added to `doc-valid-idents`.

### Lint Name

`doc_markdown`

### Reproducer

I tried this code:

```rust
#![deny(clippy::doc_markdown, reason = "example")]
//! WebAuthn.
```

I saw this happen:

```
[zack@laptop src]$ cargo clippy
    Checking example v0.1.0 (/home/zack/projects/example)
error: item in documentation is missing backticks
 --> src/lib.rs:2:5
  |
2 | //! WebAuthn.
  |     ^^^^^^^^
  |
  = help: for further information visit https://rust-lang.github.io/rust-clippy/rust-1.97.0/index.html#doc_markdown
  = note: example
note: the lint level is defined here
 --> src/lib.rs:1:9
  |
1 | #![deny(clippy::doc_markdown, reason = "example")]
  |         ^^^^^^^^^^^^^^^^^^^^
help: try
  |
2 - //! WebAuthn.
2 + //! `WebAuthn`.
  |

error: could not compile `example` (lib) due to 1 previous error
```

I expected to see this happen:

No errors.

### Version

```text
rustc 1.97.1 (8bab26f4f 2026-07-14)
binary: rustc
commit-hash: 8bab26f4f68e0e26f0bb7960be334d5b520ea452
commit-date: 2026-07-14
host: x86_64-unknown-linux-gnu
release: 1.97.1
LLVM version: 22.1.6
```
